src/overlays/actors/ovl_Toudai/ac_toudai.c

```diff
@@ -1,51 +1,298 @@
-#include "ac_toudai.h"
+#include "global.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
+#include "m_actor_dlftbls.h"
+#include "m_object.h"
+#include "c_keyframe.h"
+#include "m_field_info.h"
+#include "m_collision_bg.h"
+#include "m_rcp.h"
+#include "m_lib.h"
+#include "overlays/actors/ovl_Structure/ac_structure.h"
+
+#define THIS ((Structure*)thisx)
```

Comment:
Oh right, THIS.
I didn't include it when bootstrapping the actors since it isn't really that helpful imo, but if it is ok if we want to have it